app/src/main/java/org/fossasia/openevent/general/about/AboutEventViewModel.kt

```diff
@@ -30,8 +29,7 @@ class AboutEventViewModel(private val eventService: EventService, private val re
             return
         }
         compositeDisposable.add(eventService.getEvent(id)
-            .subscribeOn(Schedulers.io())
-            .observeOn(AndroidSchedulers.mainThread())
+        .with()
```

Comment:
`with` makes no sense

Reply:
@iamareebjamal should I change the name of extension?

Reply:
Yes

Reply:
any suggestions? @iamareebjamal

Reply:
with -> subscribeWith will that be okay or something else

Reply:
`withDefaultSchedulers`

Reply:
@iamareebjamal updated